design-documents/graph-ql/coverage/add-items-to-cart/AddSimpleProductToCart.graphqls

```diff
@@ -0,0 +1,79 @@
+type Mutation {
+    addSimpleProductsToCart(input: AddSimpleProductsToCartInput): AddSimpleProductsToCartOutput
+}
+
+input AddSimpleProductsToCartInput {
+    cartId: String!
+    cartItems: [SimpleProductCartItemInput!]!
+}
+
+input SimpleProductCartItemInput {
+    sku: String!
+    quantity: Float!
+    customizable_options:[CustomizableOptionInput!]
+}
+
+input CustomizableOptionInput {
+    id: Int!
+    value: String!
+}
+
+type AddSimpleProductsToCartOutput {
+    cart: Cart
+}
+
+type Cart {
+    id: String
+    items: [CartItemInterface]
+}
```

Comment:
Missing the cart totals, currency, updated dates and all the other information. I know that this is not in the scope but my suggestion is to add all these data to cart definition ASAP to make things clear. Btw how can a client be sure that the cart he is currently updating is the last one? The use case here is when the same cart is updated by the `end user` and also by a `back office`.